benchmarks-next/meshoptimizer/sightglass.h

```diff
@@ -0,0 +1,37 @@
+#ifndef sightglass_h
```

Comment:
I think the other benchmarks symlink this file rather than copy it, that way we can update it for all benchmarks at once easily.